### ISSUE TITLE
Cawbird 1.4.2 flatpak cannot play many embedded media files

### ISSUE THREAD

**Issue by snrkl** [IBBoard/cawbird#403]
**Describe the bug**
Clicking play on an embedded movie file,  the screenshot of the media loads full size, then immediately results in two error messages (the first is too quick to catch)

>>> ../gst/playback/gstdecodebin2.c(4694): gst_decode_bin_expose (): /GstPlayBin:video/GstURIDecodeBin:uridecodebin4/GstDecodeBin:decodebin4:
all streams without buffers

**To Reproduce**
1. Load a tweet with a relevant attachment (try https://twitter.com/emma_os/status/1447076386795966464 for example)
2. Click on the play button
3. ...

**Expected behavior**
The video file should play

**Screenshots**
![2021-10-11_18-43](https://user-images.githubusercontent.com/2288035/136760345-fd6ea24c-00d7-4197-bfa8-237f524b5023.png)
![2021-10-11_18-43_1](https://user-images.githubusercontent.com/2288035/136760334-3b37e3ae-5074-4639-b183-3d237bd2cb80.png)

**System details:**
 - OS: PoP_OS! 20.04]
 - Version: Cawbird 1.4.2
 - Package type:  PoP_OS! PoP_Shop flathub (Flatpak)
**Additional context**

Running `G_MESSAGES_DEBUG=cawbird cawbird` can give useful debug output.

>>>/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=cawbird uk.co.ibboard.cawbird
1 ~ $ /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=cawbird uk.co.ibboard.cawbird
Gtk-Message: 18:47:03.207: Failed to load module "appmenu-gtk-module"
Gtk-Message: 18:47:03.240: Failed to load module "canberra-gtk-module"
Gtk-Message: 18:47:03.241: Failed to load module "canberra-gtk-module"
(cawbird:2): cawbird-WARNING **: 18:47:04.938: load_timeline_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:04.990: load_mentions_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:05.197: load_dm_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:05.678: load_favourited_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:05.974: load_timeline_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:06.271: load_mentions_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:06.491: load_dm_tweets_done: Operation was cancelled (g-io-error-quark - 19)
(cawbird:2): cawbird-WARNING **: 18:47:06.728: load_favourited_tweets_done: Operation was cancelled (g-io-error-quark - 19)

-=[click link in tweet to video]=-

>>>** (cawbird:2): CRITICAL **: 18:47:17.794: gst_hls_demux_change_playlist: assertion 'adaptive_demux->streams != NULL' failed
(cawbird:2): cawbird-CRITICAL **: 18:47:17.794: ../ext/hls/gsthlsdemux.c(840): gst_hls_demux_handle_buffer (): /GstPlayBin:video/GstURIDecodeBin:uridecodebin0/GstDecodeBin:decodebin0/GstHLSDemux:hlsdemux0
(cawbird:2): cawbird-CRITICAL **: 18:47:17.866: ../libs/gst/base/gstbaseparse.c(1370): gst_base_parse_sink_event_default (): /GstPlayBin:video/GstURIDecodeBin:uridecodebin0/GstDecodeBin:decodebin0/GstMpegvParse:mpegvparse0
(cawbird:2): cawbird-CRITICAL **: 18:47:17.866: ../gst/playback/gstdecodebin2.c(4694): gst_decode_bin_expose (): /GstPlayBin:video/GstURIDecodeBin:uridecodebin0/GstDecodeBin:decodebin0:
all streams without buffers

**Comment by CodedOre** [comment]
Alright, made a quick test with updating the Runtime to the current 41 (which includes GStreamer 1.18.5), and the video playback worked then.

**Comment by CodedOre** [comment]
Alright, used the test build from flathub and it seems to work there.

@snrkl, if you have the time, you could test this as well:
```sh
flatpak install --user https://dl.flathub.org/build-repo/61006/uk.co.ibboard.cawbird.flatpakref
```

**Comment by snrkl** [comment]
I have time - is there a way to install it in parallel to the 1.4.2 build?

**Comment by CodedOre** [comment]
It should be a parallel install by default, as it is on the `test` branch (unlike the normal package which uses `stable`).

On that note: If you want to make sure you're running the test version, use:

```sh
flatpak run uk.co.ibboard.cawbird//test
```

**Comment by CodedOre** [comment]
> We probably could do more to explain that (but at the same time we don't want to go too far and keep bombarding the user with lots of popups as each one timeline fails when you've made too many requests!)

I was simply thinking at showing a message where the content would appear if it fails to load instead of having an infinitive loading.
But that's a topic for another issue...
